proxies/nodejs/handlers/client.ts

```diff
@@ -8,23 +8,23 @@ type ClientRequestBody = {
 }
 
 export const handleClient = async (ctx: Koa.ParameterizedContext, clients: { [key: string]: DVCClient }) => {
-  const body = <ClientRequestBody>ctx.request.body
-  if (body.clientId === undefined) {
-    ctx.status = 400
-    ctx.body = {
-      error: "Invalid request: missing clientId"
+    const body = <ClientRequestBody>ctx.request.body
+    if (body.clientId === undefined) {
```

Comment:
you can probably just return in this if statement after setting the error on the `ctx` obj so you can take the try catch out of the else statement

Reply:
the try/catch in the else is intentional for testing error cases I believe

Reply:
yeah i know i just meant you can flatten it by returning early in the if statements so it's easier to read the try catch

Reply:
ah you mean so the whole function can just be wrapped?

Reply:
```
if (statement) {
    return
}

try {

} catch (e) {

}
```
more for code cleanliness, I think it makes it easier to read if there's less indentation :)

Reply:
I getcha